core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -904,6 +900,13 @@
           + " the jar starting in the first location to the last. Supports full regex"
           + " on filename alone."),
   @Deprecated
+  GENERAL_DYNAMIC_CLASSPATHS(AccumuloVFSClassLoader.DYNAMIC_CLASSPATH_PROPERTY_NAME,
+      AccumuloVFSClassLoader.DEFAULT_DYNAMIC_CLASSPATH_VALUE, PropertyType.STRING,
+      "This property is deprecated since 2.0.0. "
+          + "A similar result can be acheived by using a custom class loader and configuring "
```

Comment:
My description of why it was deprecated is probably not a good substitute for the previous description of what it actually does. It should keep the original wording, prefixed with the "This property is deprecated since 2.0.0. " sentence. If additional explanation is needed after, perhaps we can append something like: "Built-in dynamic class loading will be removed in a future version. If you need this, consider overriding the Java system class loader with one that has this feature (https://docs.oracle.com/javase/8/docs/api/java/lang/ClassLoader.html#getSystemClassLoader--)."

Reply:
Modified the description as described and repushed.  Thanks.